man/systemd.link.xml

```diff
@@ -473,6 +473,93 @@
           </variablelist>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>Advertise=</varname></term>
+        <listitem>
+          <para>The Advertise option is used to set what speeds and modes of operation
+          (duplex mode) are advertised for auto-negotiation. The
+          supported values are:
+
+          <table>
+            <title>Supported advertise values</title>
+            <tgroup cols='5'>
+             <colspec colname='Advertise' />
+             <colspec colname='Speed' />
+             <colspec colname='Duplex Mode' />
+
+             <thead><row>
+               <entry>Advertise</entry>
+               <entry>Speed</entry>
+               <entry>Duplex Mode</entry>
+             </row></thead>
+             <tbody>
+
+               <row><entry><varname>10baset-half</varname></entry>
```

Comment:
I think this should match ethtool's output (see [ethtool.c](https://git.kernel.org/pub/scm/network/ethtool/ethtool.git/tree/ethtool.c?id=59a0467297b323f7a0ad6c0282f12696554653ff#n610)): `10baset-half` -> `10baseT/Half`.

Reply:
please use literal tag.

Reply:
OK